modules/acs-architecture-overview.adoc

```diff
@@ -9,15 +9,20 @@
 
 [NOTE]
 ====
-The architecture is slightly different when you install {product-title-short} on Kubernetes and in {ocp}.
+The architecture is slightly different when you install {product-title-short} for Kubernetes and {ocp} systems.
 However, the underlying components and the interactions between them remain the same.
 ====
 
-.{product-title-short} architecture for Kubernetes
+.{product-title-short} architecture
 
-The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 is available. Installation of Scanner V4 is optional, but provides additional benefits.
+The following graphic shows the architecture with the StackRox Scanner and Scanner V4 components. Installation of Scanner V4 is optional, but provides additional benefits.
 
-image::acs-architecture-kubernetes.png[{product-title} architecture for Kubernetes]
+[NOTE]
+====
+Sensor can communicate with Scanner in the cluster where you have installed Central. See "Architectural differences between installation on {ocp} and Kubernetes" for more information.
```

Comment:
Perhaps we should make it clear it must be enabled via Helm?

Also perhaps rather unrelated to this PR, but do we ever mention that it cannot be done via `roxctl` CLI?

Reply:
> Also perhaps rather unrelated to this PR, but do we ever mention that it cannot be done via roxctl CLI?

I don't believe we mention that

Reply:
revised, see updates